lib/main.dart

```diff
@@ -7,109 +7,302 @@ void main() {
+                        ),
+                        const Padding(padding: EdgeInsets.only(top: 10)),
+                        const Text('Chosen by Matrix'),
+                        const Padding(padding: EdgeInsets.only(top: 50)),
+                        OutlinedButton(
+                          onPressed: () {},
+                          child: const Text('Установить \n контакт'),
+                          style: OutlinedButton.styleFrom(
+                            primary: Colors.blue,
+                          ),
+                        )
+                      ],
+                    ),
+                  ),
+                  Card(
+                    child: Column(
+                      children: [
+                        const CircleAvatar(
+                          radius: 50,
+                          backgroundImage: AssetImage('assets/john.jpg'),
+                        ),
+                        const Padding(padding: EdgeInsets.only(top: 20)),
+                        const Text(
+                          'John Wick',
+                          style: TextStyle(fontWeight: FontWeight.bold),
+                        ),
+                        const Padding(padding: EdgeInsets.only(top: 10)),
+                        const Text('Someone killed \n his dog'),
+                        const Padding(padding: EdgeInsets.only(top: 50)),
+                        OutlinedButton(
+                          onPressed: () {},
+                          child: const Text('Установить \n контакт'),
+                          style: OutlinedButton.styleFrom(
+                            primary: Colors.blue,
+                          ),
+                        )
+                      ],
+                    ),
+                  ),
+                  Card(
+                    child: Column(
+                      children: [
+                        const CircleAvatar(
+                          radius: 50,
+                          backgroundImage: AssetImage('assets/geralt.png'),
+                        ),
+                        const Padding(padding: EdgeInsets.only(top: 20)),
+                        const Text(
+                          'Geralt of Rivia \n',
+                          style: TextStyle(fontWeight: FontWeight.bold),
+                        ),
+                        const Padding(padding: EdgeInsets.only(top: 10)),
+                        const Text('Struggles with C++'),
+                        const Padding(padding: EdgeInsets.only(top: 50)),
+                        OutlinedButton(
+                          onPressed: () {},
+                          child: const Text('Установить \n контакт'),
+                          style: OutlinedButton.styleFrom(
+                            primary: Colors.blue,
+                          ),
+                        )
+                      ],
+                    ),
+                  ),
```

Comment:
You should create a custom widget with parameters and use it instead of copying almost the same code 3 times.